docs/schema-loader.md

```diff
@@ -1,50 +1,83 @@
-Also, it stores internal metadata (e.g., transaction ID, record version, transaction status) for managing transaction logs and statuses when you use the Consensus Commit transaction manager.
-It is a little hard for application developers to manage the schema mapping and metadata for transactions, so we offer a tool called ScalarDB Schema Loader for creating schema without requiring much knowledge about those.
+ScalarDB has its own data model and schema that maps to the implementation-specific data model and schema. In addition, ScalarDB stores internal metadata, such as transaction IDs, record versions, and transaction statuses, to manage transaction logs and statuses when you use the Consensus Commit transaction manager.
 
-There are two ways to specify general CLI options in Schema Loader:
-  - Pass a ScalarDB configuration file and database/storage-specific options additionally.
-  - Pass the options without a ScalarDB configuration (Deprecated).
+Since managing the schema mapping and metadata for transactions can be difficult, you can use ScalarDB Schema Loader, which is a tool to create schemas that doesn't require you to need in-depth knowledge about schema mapping or metadata.
 
-Note that this tool supports only basic options to create/delete/repair/alter a table. If you want
-to use the advanced features of a database, please alter your tables with a database specific tool after creating them with this tool.
+You have two options to specify general CLI options in Schema Loader:
 
-# Usage
+- Pass the ScalarDB properties file and database-specific or storage-specific options.
+- Pass database-specific or storage-specific options without the ScalarDB properties file. (Deprecated)
 
-## Install
+{% capture notice--info %}
+**Note**
 
-The release versions of `schema-loader` can be downloaded from [releases](https://github.com/scalar-labs/scalardb/releases) page of ScalarDB.
+This tool supports only basic options to create, delete, repair, or alter a table. If you want to use the advanced features of a database, you must alter your tables with a database-specific tool after creating the tables with this tool.
+{% endcapture %}
 
-## Build
+<div class="notice--info">{{ notice--info | markdownify }}</div>
+
+## Set up Schema Loader
+
+Select your preferred method to set up Schema Loader, and follow the instructions.
+
+<div id="tabset-1">
+<div class="tab">
+  <button class="tablinks" onclick="openTab(event, 'Install', 'tabset-1')" id="defaultOpen-1">Install</button>
+  <button class="tablinks" onclick="openTab(event, 'Build', 'tabset-1')">Build</button>
+  <button class="tablinks" onclick="openTab(event, 'Docker', 'tabset-1')">Docker</button>
+</div>
+
+<div id="Install" class="tabcontent" markdown="1">
+
+You can download the release versions of Schema Loader from the [ScalarDB Releases](https://github.com/scalar-labs/scalardb/releases) page.
+</div>
```

Comment:
Since users seem to download (and later run) Schema Loader, can we change `Install` to `Download`?

Reply:
Yes, we can change `Install` to `Download`.

Reply:
Got it. I've changed this for now, even though keeping this part depends on how we resolve [this other comment](https://github.com/scalar-labs/scalardb/pull/1196/files#r1371064322).